scripts/dokka-validate.sh

```diff
@@ -2,22 +2,33 @@
 
 EXIT_TYPE="exit 1"
 MODULE=""
+GRADLE_TASK="dokkaHtmlMultiModule"
 
-SEARCH_REGEX='No documentation for com.mapbox.navigation'
-SEARCH_REGEX_EXCEPT='\.Companion ' # skip `Companion object` headers
+fileItemString=$(cat  scripts/undocumented_classes.txt |tr "\n" " ")
```

Comment:
Do we allow for the public APIs in those classes to remain undocumented?

If that's the case, could you cut a ticket to fill up the gaps?

Reply:
> Do we allow for the public APIs in those classes to remain undocumented?

We didn't have a chance to track java classes before dokka 1.4+. 

I was going to cut a ticket as this pr is landed 👍